scripts/vnet_route_check.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import json
+from swsssdk import ConfigDBConnector
+from swsssdk import SonicV2Connector
+
+
+def get_vnet_intfs():
+    config_db = ConfigDBConnector()
+    config_db.connect('CONFIG_DB')
+
+    intfs_data = config_db.get_table("INTERFACE")
+    vlan_intfs_data = config_db.get_table("VLAN_INTERFACE")
+
+    vnet_intfs = {}
+    for k, v in intfs_data.items():
```

Comment:
general comment: please use more meaningful names for loop parameters v, v1, k and k1 is not readable.